openapi_tester/type_declarations.py

```diff
@@ -40,14 +40,20 @@ def export_anything(cls, module_name: str) -> None:
     the type checker does not actually run the code.
     """
     # noinspection PyUnresolvedReferences
-    from rest_framework.response import Response
+    from rest_framework.response import Response as RFResponse
```

Comment:
I dont think this is good - instead, we should add the django rest framework mypy stubs. 

I can update our pre-commit to handle this.

Reply:
to clarify, creating a special type for response is not good.

Reply:
Ok stubs easily sounds like a better solution. I'll remove this.

Reply:
Added in af458e2b7470a176db91aa2bf2c4275e6edae0d9